packages/shared/lib/core/utils/string.ts

```diff
@@ -81,7 +66,7 @@ export function getInitials(name: string | undefined, maxChars: number): string
         .filter((n) => n)
```

Comment:
I'm getting an error that n could be null in line 67 when accessing n[0] so I did some googling and found some possible solutions
```suggestion
        .filter((n): n is RegExpMatchArray => n !== null)
```
or you could use a flatMap `.flatMap((n) => n ? [n[0]] : [])` to do the same thing without needing to make use of type guard

sources:
https://stackoverflow.com/a/51577579
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap#for_adding_and_removing_items_during_a_map

Reply:
Otherwise, LGTM